pkg/errors/rolloutblock.go

```diff
@@ -0,0 +1,45 @@
+package errors
+
+import (
+	"fmt"
+)
+
+type InvalidRolloutBlockOverrideError struct {
+	RolloutBlockName string
+}
+
+func (e InvalidRolloutBlockOverrideError) Error() string {
+	return fmt.Sprintf("rollout block with name %s does not exist",
+		e.RolloutBlockName)
+}
+
+func (e InvalidRolloutBlockOverrideError) ShouldRetry() bool {
+	return false
+}
+
+func (e InvalidRolloutBlockOverrideError) ShouldBroadcast() bool {
+	return false
+}
+
+func NewInvalidRolloutBlockOverrideError(invalidRolloutBlockName string) InvalidRolloutBlockOverrideError {
+	return InvalidRolloutBlockOverrideError{invalidRolloutBlockName}
+}
+
+type RolloutBlockError string
+
+func (e RolloutBlockError) Error() string {
+	return string(e)
+}
+
+func (e RolloutBlockError) ShouldRetry() bool {
+	return false
+}
+
+func (e RolloutBlockError) ShouldBroadcast() bool {
+	return false
```

Comment:
I think that all the errors here should be broadcastable (as in, they will be also sent as an event), to be in line with the majority of our other errors.